app/views/embassies/index.html.erb

```diff
@@ -2,10 +2,11 @@
 <% page_class "worldwide-organisations embassies-index" %>
 
 <header class="block headings-block">
- <div class="inner-block floated-children">
-   <%= render partial: 'shared/heading',
-             locals: { heading: "Find a British embassy, high commission or consulate",
-                       type: "Worldwide" } %>
+  <div class="inner-block floated-children">
+    <%= render "govuk_publishing_components/components/title", {
+      context: "Worldwide",
+      title: "Find a British embassy, high commission or consulate",
+    } %>
```

Comment:
👍🏼 Not a blocker but noting a visual change

![image](https://user-images.githubusercontent.com/71266765/109803807-68404080-7c19-11eb-8db3-2e925978ad27.png)